gcp-observability/src/main/java/io/grpc/gcp/observability/interceptors/ConfigFilterHelper.java

```diff
@@ -16,51 +16,28 @@
 
 package io.grpc.gcp.observability.interceptors;
 
+import static com.google.common.base.Preconditions.checkNotNull;
+
 import com.google.auto.value.AutoValue;
 import com.google.common.annotations.VisibleForTesting;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableSet;
 import io.grpc.Internal;
-import io.grpc.MethodDescriptor;
 import io.grpc.gcp.observability.ObservabilityConfig;
 import io.grpc.gcp.observability.ObservabilityConfig.LogFilter;
-import io.grpc.observabilitylog.v1.GrpcLogRecord.EventType;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.logging.Level;
-import java.util.logging.Logger;
 
 /**
  * Parses gRPC GcpObservability configuration filters for interceptors usage.
  */
 @Internal
 public class ConfigFilterHelper {
-
-  private static final Logger logger = Logger.getLogger(ConfigFilterHelper.class.getName());
-
   public static final FilterParams NO_FILTER_PARAMS
       = FilterParams.create(false, 0, 0);
-  public static final String globalPattern = "*";
 
   private final ObservabilityConfig config;
-  @VisibleForTesting
-  boolean methodOrServiceFilterPresent;
-  // Flag to log every service and method
-  @VisibleForTesting
-  Map<String, FilterParams> perServiceFilters;
-  @VisibleForTesting
-  Map<String, FilterParams> perMethodFilters;
-  @VisibleForTesting
-  Set<EventType> logEventTypeSet;
 
   @VisibleForTesting
```

Comment:
`getInstance` is public and just calls this constructor. You can just make this private and have the tests call `getInstance`

Reply:
Done